### DISCUSSION TITLE
CB7695EB-5BC5-B8BF-3687-9A610E5CC949

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#45] [General]
# CB7695EB-5BC5-B8BF-3687-9A610E5CC949

Adds two asteroids that have made close approaches to Earth in 2020

https://celestia.mobi/addon?item=CB7695EB-5BC5-B8BF-3687-9A610E5CC949&lang=zh_CN

**Comment by 13940951572** [comment]
你好

**Comment by 13940951572** [comment]
希望能出一个近地小行星2023 FW13
